src/Model/Filter/Base.php

```diff
@@ -111,7 +112,7 @@ public function field()
         }
 
         $repository = $this->manager()->getRepository();
-        if (!method_exists($repository, 'aliasField')) {
+        if (!$repository instanceof Table) {
```

Comment:
Could this be a BC break for people implementing their own RepositoryInterface that does not extend Table?
Should this maybe be "or"?